package.json

```diff
@@ -54,6 +55,7 @@
     "backbone": "1.2.1 - 1.3.x",
     "chai": "3.4.0",
     "chai-jq": "0.0.9",
+    "easy-sauce": "^0.3.5",
```

Comment:
would it be ideal to lock down the version in case of something randomly breaking